standard_releases.md

```diff
@@ -1,59 +1,206 @@
+[Semantic Versioning](https://semver.org/), where each part of a version number
+has significance to users and is used by them to understand the differences
+between versions.
+
+These version numbers are applied in the main BODS repository
+[github.com/openownership/data-standard](https://github.com/openownership/data-standard)
+as a naming convention for release branches, for example:
+
+* [github.com/openownership/data-standard/tree/v0-0-3](https://github.com/openownership/data-standard/tree/v0-0-3)
+* [github.com/openownership/data-standard/tree/v0-1](https://github.com/openownership/data-standard/tree/v0-1)
+
+These releases are published by enabling the release branch in ReadTheDocs, the
+"latest" tag in ReadTheDocs is manually changed to point at the latest version.
+
+**Note:** our release branch naming scheme is "vMAJOR-MINOR-PATCH"
+
+**Note:** we always include trailing zeroes in our version numbers. e.g. "v0-1-0" is
+preferred over "v0-1" because someone might interpret the latter as the latest
+version of v0.1.
+
+## Translations and releases
+
+Our standard assumption for translations is that they are generally
+**asynchronous with development**. That means that changes to the standard or
+documentation do not need to wait for translation before they can be released,
+they should be planned and scheduled as appropriate. For small changes, this
+most likely means batching and performing translations when a suitable number
+are required. For larger changes, particularly new releases, we're more likely
+to want to have all supported translations in place before releasing.
+
+## Release processes
+
+How exactly we perform a release depends on what is being released and which
+versions the changes affect. The sections below detail specific scenarios, but
+the following general questions always apply:
+
+1. Has the schema or any normative documentation changed?
+2. Does text need translating?
+3. Do the changes affect any existing, released versions?
+
+### A new release
+
+Preparing a new release is the most straightforward scenario, though not the
+least work! The general process is:
+
+* Decide on the set of changes that will be released. Some of these will have
+  likely already been merged in the course of normal development, but they may
+  need additional development, examples adding, etc.
+* Review, test and merge those changes into the `master` branch.
+* Depending on the other ongoing work, either call a "freeze" on the master
+  branch, whereafter merging changes that are not going to be part of the
+  release cannot be merged, or create a new release branch from master. A
+  release freeze is preferred, to reduce the complexity of incorporating any
+  release fixes back into master, but this can be weighed against the impediment
+  to other development on a case-by-case basis.
+* Begin translation of the new release (see [translations](/translations)) and
```

Comment:
* Create a new branch from `master` for the translation work. eg. `v0-2-0-translation`
* Begin translation of the new release (see [translations](/translations))
* During translation there may be a need to fix issues that exist on the master branch. If they are outside `schema` and `docs` they can be developed on a branch taken from `master` and merged back in when completed.
If the changes affect `schema` and/or `docs` then they may, in turn, affect the translation. The process for making these changes is to first create an issue in Github. Then create a new branch that uses the issue number, eg. `297-concept-translation`. When the branch is ready to be merged into `master`, pull the strings from Transifex and push them to the translation branch eg `v0-2-0-translation`. Now merge the branch (`297-concept-translation`) into `master` before pushing the updated docs to Transifex. Check for any changes in Transifex and use the translation branch (`v0-2-0-translation`) to replace any translations that have been overwritten.
^^^ this could probably do with a diagram =0)